about.html

```diff
@@ -13,56 +13,75 @@
 
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
+             <header>
+            <img class = "lambdaBlack" src = C:\Users\Mitch\Desktop\web-Sprint-Challenge-User-Interface-and-Git\img\lambda-black.png>
+            <nav>
+                <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+            </nav>
+        </header>
+        <img src = C:\Users\Mitch\Desktop\web-Sprint-Challenge-User-Interface-and-Git\img\jumbo-about.png>
+  <div class = "aboutText"> 
+        <h3>About</h3>
+
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
```

Comment:
Please be mindful of spaces and indentation, try to keep your code organize to help with readability. its a good habit to start developing early